default.json

```diff
@@ -41,7 +41,11 @@
       "semanticCommitType": "fix"
     },
     {
-      "excludePackageNames": ["@koa/cors", "@seek/ie-logging"],
+      "excludePackageNames": [
+        "@koa/cors",
+        "@seek/ie-logging",
+        "@seek/indie-api-types"
```

Comment:
```suggestion
        "@seek/indie-api-types",
        "@seek/indie-mocks"
```

Reply:
👍 I omitted because it looked like there was one consumer, but doesn't hurt